### ISSUE TITLE
quickFix for missing translation not shown

### ISSUE THREAD

**Issue by gb5256** [kalessil/yii2inspections#34]
HI,
your plugin does show corretcly that the translation for "jolly" is missing (see screenshot and notice that it found the translation for "welcome".)
![screenshot 2019-01-01 at 22 40 27](https://user-images.githubusercontent.com/324658/50576639-63538c80-0e16-11e9-9ba5-5f5f973d76f8.jpg)
But it does not show the quickfix for adding the translation to the translation file:
![screenshot 2019-01-01 at 22 44 13](https://user-images.githubusercontent.com/324658/50576659-c9d8aa80-0e16-11e9-9ebe-db5a46e3af35.jpg)

Any Idea ?
I am using the advanced template, and the translation files are inside "frontend". I have also tried the same with moving the translation folder into the root of the app, but no difference.

gb5256

**Comment by kalessil** [comment]
The plugin searches translations in files matching some naming conventions: https://github.com/kalessil/yii2inspections/blob/master/src/main/com/kalessil/phpStorm/yii2inspections/codeInsight/TranslationKeysIndexer.java#L86

e.g. `[any-path=here]/translations/en.php`, `[any-path=here]/translations/EN.php` or `[any-path=here]/translations/en-us.php` will be recognized as translation files.

If the convention is already used, then I need some details regarding translation files full path and structure.

**Comment by gb5256** [comment]
Hi, thanks for your swift reply.
I have now created it as you suggested. And for the case that it might be related to the yii2-advanced folder structure, I have also added it to the root. So I have now two translation folders, just to be sure. see screenshot. Still no quickfix.
![screenshot 2019-01-01 at 23 10 43](https://user-images.githubusercontent.com/324658/50576833-b3345280-0e1a-11e9-9915-78cd36f8a93e.jpg)
